data/microos/butane/script

```diff
@@ -79,28 +82,64 @@ test_prepare() {
     echo "combustion: test_prepare function ran OK" > /dev/kmsg
 }
 
+can_encrypt() {
+	if [ -c "/dev/tpm0" ] && command -v disk-encryption-tool >/dev/null; then
+		return 0
+	else
+		return 1
+	fi
```

Comment:
A nitpick.  The if/else is redundant here:

```suggestion
	[ -c "/dev/tpm0" ] && command -v disk-encryption-tool >/dev/null
```

Reply:
Added, thanks!